distage/distage-framework/src/main/scala/izumi/distage/framework/services/ConfigLoader.scala

```diff
@@ -29,10 +29,11 @@ object ConfigLoader {
     *   - $roleName-reference-dev.conf
     */
   class LocalFSImpl(
-    logger: IzLogger,
-    primaryConfig: Option[File],
-    roleConfigs: Map[String, Option[File]],
-  ) extends ConfigLoader {
+                     logger: IzLogger,
+                     primaryConfig: Option[File],
+                     roleConfigs: Map[String, Option[File]],
+                     configOverride: AppConfig => AppConfig = identity
```

Comment:
@neko-kai @pshirshov 
Is this looks good? I don't wanna add new method here, because it's redundant.

Reply:
@Caparow This parameter is redundant given `ConfigLoader.map` method here - https://github.com/7mind/izumi/pull/807/files#diff-5c8b9d105825c5921af472f1d4cce8b4R17

Reply:
@Caparow Also, could you update the comment with default config locations here - https://github.com/7mind/izumi/pull/807/files#diff-5c8b9d105825c5921af472f1d4cce8b4R25 ?

Reply:
> This parameter is redundant given ConfigLoader.map
I've miss this method ._.
Will update this